src/main/java/lotto/domain/Lotto/AnswerLotto.java

```diff
@@ -0,0 +1,50 @@
+package lotto.domain.Lotto;
+
+import java.util.List;
+
+public class AnswerLotto {
+  private List<Integer> answerNumbers;
+  private int bonusNumber;
+  
+  public AnswerLotto(List<Integer> answerNumbers, Integer bonusNumber) {
+    validateLottoSize(answerNumbers);
+    validateLottoHasDuplicate(answerNumbers);
+    validateLottoNumbersRange(answerNumbers);
+    this.answerNumbers = answerNumbers;
+    this.bonusNumber = bonusNumber;
+  }
+  
+  private void validateLottoSize(List<Integer> numbers) {
+    if (numbers.size() != 6) {
+      throw new IllegalArgumentException("정답 로또 번호가 6자리가 아닙니다.");
+    }
+  }
+  
+  private void validateLottoHasDuplicate(List<Integer> numbers) {
+    if (hasDuplicate(numbers)) {
+      throw new IllegalArgumentException("로또 번호에 중복된 번호가 있습니다.");
+    }
+  }
+  
+  private boolean hasDuplicate(List<Integer> numbers) {
+    return numbers.stream().distinct().count() != 6;
+  }
```

Comment:
```suggestion
private boolean hasDuplicateNumber(final List<Integer> baseballs) {
    return baseballs.stream()
            .distinct()
            .count() != LOTTO_SIZE;
}
```
- `Dot(.)`은 1줄에 1개만 쓰는게 가독성에 좋습니다
- 매직넘버 X